raiwidgets/requirements-dev.txt

```diff
@@ -13,6 +13,8 @@ ml-wrappers>=0.4.0
 
 # Jupyter dependency that fails with python 3.6
 pywinpty==2.0.2; python_version <= '3.6' and sys_platform == 'win32'
+# fix for recent joblib release and windows python 3.7 builds
```

Comment:
not sure if I should just pin the requirements.txt to make sure we don't have this issue in prod or that might be too heavy-handed and I should go with the less intrusive fix to only resolve our build failures (who knows what the ramifications might be for the more heavy-handed fix... and relatively few people probably use this on python 3.7 w/ windows anyway)

Reply:
Python 3.7 reached EOL yesterday. Perhaps it's time to drop it 🙂

https://peps.python.org/pep-0537/